src/components/calcite-shell/calcite-shell.tsx

```diff
@@ -54,18 +54,37 @@ export class CalciteShell {
   }
 
   renderContent(): VNode {
+    return !!this.contentBehind ? this.renderContentBehind() : this.renderContentInline();
+  }
+
+  renderContentBehind(): VNode {
     return (
       <div
         class={{
           [CSS.content]: true,
-          [CSS.contentBehind]: !!this.contentBehind
+          [CSS.contentBehind]: true
```

Comment:
@asangma this doesn't look right.

Reply:
I can add that back in.
I figured we didn't need since it's inside a function that's only called on that condition, but I can see that it'd be more watertight to keep that conditional in there.

Reply:
yeah if its not needed then it can be removed I guess. Otherwise it should remain a conditional modifier.

Reply:
Updated.